<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleHistoryDetail</class>
 <widget class="QWidget" name="SampleHistoryDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1696</width>
    <height>752</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1696</width>
    <height>752</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1696</width>
    <height>752</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,6">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1670</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1670</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
background: #37ADBA;
border-radius: 10px 10px 0px 0px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>341</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lab_title">
        <property name="styleSheet">
         <string notr="true">font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FFFFFF;
line-height: 42px;</string>
        </property>
        <property name="text">
         <string>title</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>340</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1670</width>
       <height>161</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1670</width>
       <height>161</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2
{
background: #FFFFFF;
border-radius: 10px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lab_num">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #333333;
line-height: 42px;</string>
        </property>
        <property name="text">
         <string>样本编号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_num_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>380</width>
          <height>58</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>380</width>
          <height>58</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);
font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
        </property>
        <property name="text">
         <string>12345</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>259</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lab_location">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #333333;
line-height: 42px;</string>
        </property>
        <property name="text">
         <string>样本位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_location_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>380</width>
          <height>58</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>380</width>
          <height>58</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #F2F7FB;
border-radius: 29px;
border: 1px solid rgba(170,189,212,0.6);
font-family: PingFang SC;
font-weight: bold;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
        </property>
        <property name="text">
         <string>12345</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>259</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget">
        <property name="styleSheet">
         <string notr="true">QWidget#widget_table
{
background: #FFFFFF;
border-radius: 10px;
}
QTableWidget#tableWidget
{
background: #FFFFFF;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
border-radius: 10px;
color: #333333;
line-height: 42px;
}
QTableWidget::item {
    text-align: center;
}
QHeaderView {
    font-family: PingFang SC;
	font-weight: 500;
	font-size: 22px;
	color: #6D7E92;
	line-height: 42px;
}

QHeaderView::section {
    background-color: #EFF0F2;
	border-top: 1px solid #E1EBF6;
	border-left: none;
	border-bottom: 1px solid #E1EBF6;
	border-right: none;
	line-height: 42px;
}
QTableWidget::item:selected {
    background: #D4F2F5;
	border-top: none;
	border-left: none;
	border-bottom: 1px solid #22A4B3;
	border-right: none;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
